src/model/differ.js

```diff
@@ -168,9 +191,23 @@ export default class Differ {
 	 * the position on which the change happened. If a position {@link module:engine/model/position~Position#isBefore is before}
 	 * another one, it will be on an earlier index in the diff set.
 	 *
+	 * Because calculating diff is a costly operation, the result is cached. If no new operation was buffered since the
+	 * previous {@link #getChanges} call, the next call with return the cached value.
+	 *
+	 * @params {Boolean} [includeChangesInGraveyard=false] If set to `true`, also changes that happened in graveyard root will be returned.
 	 * @returns {Array.<Object>} Diff between old and new model tree state.
 	 */
-	getChanges() {
+	getChanges( includeChangesInGraveyard = false ) {
```

Comment:
I would use object `option`  with `includeChangesInGraveyard` as property instead of boolean   `includeChangesInGraveyard`. It would be safer for the future if we will introduce more properties and `getChanges( { includeChangesInGraveyard: true } )` is more readable than `getChanges( true )`.